docs/DEVGUIDE.md

```diff
@@ -264,7 +275,7 @@ Next, we'll register a service broker with the catalog.  To do this, we'll
 create a new [`Broker`](../contrib/examples/walkthrough/ups-broker.yaml)
 resource:
 
-    $ kubectl create -f contrib/examples/walkthrough/ups-broker.yaml
+    $ ./kubectl16 create -f contrib/examples/walkthrough/ups-broker.yaml
```

Comment:
yes. I haven't seen docs that tell users to download a 1.6 beta version of `kubectl`